scripts/docker-integration-tests/aggregator/m3aggregator.yml

```diff
@@ -0,0 +1,277 @@
+logging:
```

Comment:
:(

Reply:
Yeah this config is insane.  We’ll have to improve over time.